src/core/engines/Cesium/Feature/Box/hooks/scalePoint.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  CallbackProperty,
+  Cartesian3,
+  Color,
+  JulianDate,
+  Matrix4,
+  PolylineDashMaterialProperty,
+  PositionProperty,
+  Quaternion,
+  ScreenSpaceEventHandler,
+  ScreenSpaceEventType,
+  TranslationRotationScale,
+} from "cesium";
+import { useCallback, useMemo, useState, useEffect, useRef } from "react";
+import { useCesium } from "resium";
+
+import { EventCallback } from "@reearth/core/Map";
```

Comment:
```suggestion
import type { EventCallback } from "@reearth/core/Map";
```

If possible, if you are importing a type, please add the type keyword. This may improve compile speed since you can remove it at compile time without the need for type inference.

Reply:
I've fixed it
[fix: use type import](https://github.com/reearth/reearth-web/pull/377/commits/6a8c05be0283c07328447a8d6607cb5365a0d5eb)